libs/blocks/section-metadata/section-metadata.js

```diff
@@ -56,6 +56,7 @@ export async function handleStyle(text, section) {
   const styles = text.split(', ').map((style) => style.replaceAll(' ', '-'));
   const sticky = styles.find((style) => style === 'sticky-top' || style === 'sticky-bottom');
   if (sticky) await handleStickySection(sticky, section);
+  if (styles.includes('masonry')) styles.push('masonry-up');
```

Comment:
Why not just add `masonry-up` to the section metadata styles?

Reply:
I thought about that, but I feel like "masonry up" is not what anyone would call it and I'm trying to keep it author-friendly.  So I really wanted to call it "masonry".  And 1 line of JS saves us a bunch of lines of CSS.